test/e2e/manifest.yaml

```diff
@@ -0,0 +1,29 @@
+# Test manifest for https://github.com/kubernetes/kubernetes/tree/master/test/e2e/storage/external
+ShortName: manila
+StorageClass:
+  FromExistingClassName: standard-csi
+SnapshotClass:
+  FromName: true
+DriverInfo:
+  Name: manila.csi.openstack.org
+  SupportedSizeRange:
+    Min: 1Gi
+    Max: 16Ti
+  SupportedFsType:
+    xfs: {}
+    ext4: {}
+  SupportedMountOption:
+    dirsync: {}
```

Comment:
Again, not sure if `mount -o dirsync` is supported on NFS. Goal of this field is to have an innocent mount option that the test can apply during mount and is then visible in /proc/mounts to test if / how the driver supports mount options.

Reply:
ack, I'll remove `SupportedMountOption`.

Reply:
Hold on, I think testing mount options are useful. You just need to use one that works for NFS and is relatively harmless, like `soft` or `nolock` (it's used only for testing)